minidump-stackwalk/http_symbol_supplier.cc

```diff
@@ -236,6 +236,20 @@ namespace {
   string JoinURL(CURL* curl, const string& url, const string& sub) {
     return url + "/" + URLEncode(curl, sub);
   }
+
+  string GetCodeIDAndFileQueryString(CURL* curl, const CodeModule* module) {
+      string code_file = PathnameStripper::File(module->code_file());
+      if (code_file.empty()) {
+        return "";
+      }
+
+      string code_id = PathnameStripper::File(module->code_identifier());
+      if (code_id.empty()) {
+        return "";
+      }
+
+      return "?code_file=" + URLEncode(curl, code_file) + "&code_id=" + URLEncode(curl, code_id);
```

Comment:
I'm assuming you're actually referring to line 316'ish where it appends the query string. 
THis function here looks like a plain function that returns a string. 

But isn't `full_url` a regular string?